explorer/explorertypes.go

```diff
@@ -343,3 +325,74 @@ type NewMempoolTx struct {
 	Time int64
 	Hex  string
 }
+
+// ExtendedChainParams represents the data of ChainParams
+type ExtendedChainParams struct {
+	Params        *chaincfg.Params
+	AddressPrefix []AddrPrefix
+}
+
+// AddrPrefix represent the address name it's prefix and description
+type AddrPrefix struct {
+	Name        string
+	Prefix      string
+	Description string
+}
+
+// AddressPrefixes generates an array AddrPrefix by using chaincfg.Params
+func AddressPrefixes(params *chaincfg.Params) []AddrPrefix {
+	var Descriptions = []string{"First 2 letters of a P2PK address",
+		"First 2 letters of a P2PKH address",
+		"First 2 letters of an Edwards P2PKH address",
+		"First 2 letters of a secp256k1 Schnorr P2PKH address",
+		"First 2 letters of a P2SH address",
+		"First 2 letters of a WIF private key",
+		"BIP32 hierarchical deterministic extended key magics",
+		"BIP32 hierarchical deterministic extended key magics",
+	}
+
+	addrPrefix := make([]AddrPrefix, 0, len(Descriptions))
+
+	var Name = []string{"PubKeyAddrID",
+		"PubKeyHashAddrID",
+		"PKHEdwardsAddrID",
+		"PKHSchnorrAddrID",
+		"ScriptHashAddrID",
+		"PrivateKeyID",
+		"HDPrivateKeyID",
+		"HDPublicKeyID",
+	}
```

Comment:
I'm not convinced we need the `Name` column since the description is more clear than the variable name, and if you want to know the variable name for any reason, you are probably already looking at the code.

Reply:
By keeping it for now, the table columns will continue to align up.